compiler-cli/src/main.rs

```diff
@@ -158,6 +158,13 @@ pub struct NewOptions {
         default_value = "lib"
     )]
     pub template: new::Template,
+
+    #[structopt(
+        long,
+        possible_values = &new::Docker::VARIANTS,
+        case_insensitive = true,
+    )]
+    pub docker: Option<Option<new::Docker>>,
```

Comment:
Option Option allows the command to be used as `--docker` without any argument or `--docker default/slim/alpine`